### PR TITLE
update equationgrapher

### PR DESCRIPTION
**Pull request by thacuber2a03**
update mod-version and fix usage of deprecated `command_view:enter`

also invert mouse zoom

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -376,7 +376,7 @@
     {
       "description": "Graphs y=x equations.",
       "version": "0.1",
-      "remote": "https://github.com/ThaCuber/equationgrapher?raw=1:bd7056b5d4a6f6b98365e1a5e3e78e29ed843d13",
+      "remote": "https://github.com/thacuber2a03/equationgrapher?raw=1:7b17d8de47ab5c40019c1910a303340b5176e105",
```
Comment (Guldoman) [inline manifest.json:379]:
```suggestion
      "remote": "https://github.com/thacuber2a03/equationgrapher:7b17d8de47ab5c40019c1910a303340b5176e105",
```

File: manifest.json [hunk 2]
```diff
@@ -376,7 +376,7 @@
     {
       "description": "Graphs y=x equations.",
       "version": "0.1",
```
Comment (Guldoman) [inline manifest.json:378]: The version should be bumped.

Comment (thacuber2a03) [reply manifest.json:378]: eef I forgot

**Comment by Guldoman** [comment]
Looks good, thanks!

**Pull request merged**